mesondev.sh

```diff
@@ -33,6 +33,7 @@ python -c "from scipy.sparse import csgraph as s; s.test()"
 # python -c "from scipy import optimize as s; s.test()" # relies on sparse.linalg
 # python -c "from scipy import stats as s; s.test()" # relies on spatial
 # python -c "from scipy import special as s; s.test()" # relies on spatial
+# python -c "from scipy import spatial as s; s.test()" # relies on scipy._lib.messagestream
```

Comment:
I checked `_lib` and it seems like, build for `messagestream` isn't added yet.

Reply:
Ah yes, that's one of the first submodules I did, and I decided to postpone dealing with `messagestream` because of its weird config-check-writes-a-file plus Cython. If we need it now, I can revisit that (but probably not today).

Reply:
For compatibility with `setup.py`, keeping the config header file seems best. Docs: https://mesonbuild.com/Configuration.html#configuration

Reply:
> (but probably not today).

No problem. I can work on a different module until `messagestream` is added.

Reply:
`messagestream` is now available in the `meson` branch, see commit 9b26ed6f85

Reply:
I also rebased on upstream master.